### DISCUSSION TITLE
multiple file upload(field) - set size limitation per file

### DISCUSSION THREAD

**Discussion by sadeghhosseini** [expressjs/multer#1147] [Q&A]
Is it possible to specify limit per file when we are using `fields`  to upload multiple files.
e.g.

```
const upload = multer({ dest: 'uploads/' })
const cpUpload = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }])
```
What I mean is that for instance in the example above, is it possible to set file size limit on `avatar` and `gallery` separately?
I tried using `fileFilter` but the second parameter of `fileFilter` which is `file` does not provide file size.

**Comment by moriseif** [comment]
Are you find a way sadegh?
same question for me ...
